### ISSUE TITLE
Outed Mixins Issue with Core. 

### ISSUE THREAD

**Issue by StoneLegion** [Ordinastie/MalisisCore#162]
https://pastebin.com/jBVNAAvq

You shipping with 0.7.4 and I think latest is 0.7.11 :)

Thanks.

**Comment by KiraLinux** [comment]
I have the same
Mixin config mixins.sgcraft.json requires mixin subsystem version 0.7.10 but 0.7.4 was found. The mixin config will not be applied.

https://gist.github.com/KiraLinux/4b1c08868d49c5612a413313feda425e

**Comment by Dragnier** [comment]
Yep, same issue here. https://github.com/AlmuraDev/SGCraft/issues/45
